libs/clients/rsk/company-registry/README.md

```diff
@@ -1,6 +1,6 @@
-# Clients RSK Company Info
+# Clients Company Registry
 
-This library was generated with [Nx](https://nx.dev).
+This is a client for Company Registry, providing access to endpoints for fetching Company information by nationalId as well as Company information search.
 
```

Comment:
`nationalId` vs `national id` in generalized text? :thinking:

Reply:
@kksteini good catch, fixed